juriscraper/pacer/rss_feeds.py

```diff
@@ -116,8 +116,8 @@ class PacerRssFeed(DocketReport):
     #   [Scheduling Order] (<a href="https://ecf.mad.uscourts.gov/doc1/
     #   09518690740?caseid=186403&de_seq_num=98">39</a>)
     # We use three simple matches rather than a complex one with three groups.
-    document_number_regex = re.compile(r'">(\d+)</a>')
-    doc1_url_regex = re.compile(r'href="(.*/doc1/.*)"')
+    document_number_regex = re.compile(r"[\"|\'] ?>(\d+)</a>")
+    doc1_url_regex = re.compile(r"href=[\"|\'](.*/doc1/.*)[\"|\']")
```

Comment:
I think I'd put these back and not change them unless there's something that broke? Looks like your intent was to make it robust to people using single quotes in their HTML, but I think your regexes introduce an additional character too (the pipe isn't needed in a character class), and they add complexity that's not needed. Unless something changed, YAGNI?

Reply:
They were necessary changes.

Reply:
A handful of <a> tags were quoted with single and the others double.  I don't know how this could've worked before without it.  Additionally one or two had an errant space after quotes.